modules/chat/client-react/containers/withUuid.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import uuid from 'uuid';
 
-import { getItem, setItem } from '@gqlapp/core-common';
+import { getItem, setItem } from '@gqlapp/core-common/clientStorage';
```

Comment:
```suggestion
import { clientStorage } from '@gqlapp/core-common';
```
then use `clientStorage.getItem`, `clientStorage.setItem`